engine/src/main/java/org/destinationsol/game/ship/SolShip.java

```diff
@@ -378,24 +378,24 @@ private void throwAllLoot(SolGame game) {
     }
 
     private void throwLoot(SolGame game, SolItem item, boolean onDeath) {
-        Vector2 lootSpeed = new Vector2();
+        Vector2 lootVelocity = new Vector2();
         float speedAngle;
```

Comment:
`velocityAngle`? maybe look for `speedAngle` even in other files, in case I missed it.

Reply:
Not sure with this, from I understand what velocity consists of speed and direction, which is an angle in DestSol, so speedAngle suits better.
Maybe @eviltak can bring in some light into this?